### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by anthonyqzhu**
Adjust servlet code to use helper methods in order to make code more readable

### CODE REVIEW

**Comment by yjennywang** [comment]
This depends on the last pull request, right?

**Comment by anthonyqzhu** [comment]
> This depends on the last pull request, right?

Yea this one can wait until the auth fix and the last one are approved

**Comment by yjennywang** [comment]
> > This depends on the last pull request, right?
> 
> Yea this one can wait until the auth fix and the last one are approved

okay, thanks!

Let me know when you want me to look at it again.

**Comment by apoorvadhanala** [comment]
> > > This depends on the last pull request, right?
> > 
> > 
> > Yea this one can wait until the auth fix and the last one are approved
> 
> okay, thanks!
> 
> Let me know when you want me to look at it again.

@anthonyqzhu do you want me to look at this now or is it too early? can't tell for sure!

**Comment by anthonyqzhu** [comment]
> > > > This depends on the last pull request, right?
> > > 
> > > 
> > > Yea this one can wait until the auth fix and the last one are approved
> > 
> > 
> > okay, thanks!
> > Let me know when you want me to look at it again.
> 
> @anthonyqzhu do you want me to look at this now or is it too early? can't tell for sure!

Not right now, will re request when its ready!

**Comment by yjennywang** [comment]
Can you try a git pull --rebase origin/master to make sure it's up to date?

(Maybe git checkout -b copy_branch first to make a copy and then try, in case that gets hard and doesn't work well with this branch)

**Comment by anthonyqzhu** [comment]
Pulled and rebased from master so it should be up to date - the commits might be hard to read now so maybe just review files changed? Anyways this branch is ready for review now

File: src/main/java/com/google/sps/servlets/CalendarServlet.java [hunk 1]
```diff
@@ -43,11 +43,13 @@
  * Servlet responsible for returning the link to the shared family calendar
 */
 public class CalendarServlet extends AbstractAppEngineAuthorizationCodeServlet {
+
+  static final String APP_NAME = "Household";
```
Comment (sidharthp) [inline src/main/java/com/google/sps/servlets/CalendarServlet.java]: Is this variable used anywhere?

**Review by sidharthp — APPROVED** [review]

File: src/main/java/com/google/sps/servlets/CalendarServlet.java (hunk 1 above)
Comment (anthonyqzhu) [reply src/main/java/com/google/sps/servlets/CalendarServlet.java]: Oops I think it's used in a different area, will remove

**Pull request merged**